Makefile

```diff
@@ -170,6 +176,16 @@ IMG = quay.io/crcont/crc-e2e:$(CRC_E2E_IMG_VERSION)
 containerized_e2e: clean
 	$(CONTAINER_RUNTIME) build -t $(IMG) -f images/build-e2e/Dockerfile .
 
+#  Build the container image for integration
+.PHONY: containerized_integration
+containerized_integration:
+ifndef CRC_INTEGRATION_IMG_VERSION
+CRC_INTEGRATION_IMG_VERSION=v$(CRC_VERSION)-$(COMMIT_SHA)
+endif
```

Comment:
mm the intention is to build something like: 

CRC_INTEGRATION_IMG_VERSION=dev make containerized_integration, so I can have a dev image for testing purposes

Reply:
I understand, but I think `?=` achieves a similar purpose

> There is another assignment operator for variables, `?=`. This is called a conditional variable assignment operator, because it only has an effect if the variable is not yet defined. This statement:
```
FOO ?= bar
```
is exactly equivalent to this (see The origin Function):
```
ifeq ($(origin FOO), undefined)
  FOO = bar
endif
```
Note that a variable set to an empty value is still defined, so `?=` will not set that variable.